packages/web-components/fast-components/src/tab/tab.styles.ts

```diff
@@ -42,122 +49,119 @@ export const tabStyles: (
     definition: FoundationElementDefinition
 ) =>
     css`
-    ${display("inline-flex")} :host {
-        box-sizing: border-box;
-        font-family: ${bodyFont};
-        font-size: ${typeRampBaseFontSize};
-        line-height: ${typeRampBaseLineHeight};
-        height: calc(${heightNumber} * 1px);
-        padding: calc(${designUnit} * 5px) calc(${designUnit} * 4px);
-        color: ${neutralForegroundHint};
-        fill: currentcolor;
-        border-radius: calc(${controlCornerRadius} * 1px);
-        border: calc(${strokeWidth} * 1px) solid transparent;
-        align-items: center;
-        justify-content: center;
-        grid-row: 1;
-        cursor: pointer;
-    }
+        ${display("inline-flex")} :host {
+            box-sizing: border-box;
+            font-family: ${bodyFont};
+            font-size: ${typeRampBaseFontSize};
+            font-weight: 400;
+            line-height: ${typeRampBaseLineHeight};
+            height: calc(${heightNumber} * 1px);
+            padding: 0 calc((6 + (${designUnit} * 2 * ${density})) * 1px);
+            background: ${neutralFillStealthRest};
+            color: ${neutralForegroundRest};
+            fill: currentcolor;
+            border-radius: calc(${controlCornerRadius} * 1px);
+            border: calc(${strokeWidth} * 1px) solid transparent;
+            align-items: center;
+            justify-content: center;
+            cursor: pointer;
+            outline: none;
+        }
 
-    :host(:hover) {
-        color: ${neutralForegroundRest};
-        fill: currentcolor;
-    }
+        :host(:not(.disabled):hover),
+        :host(.vertical:not(.disabled):hover) {
+            background: ${neutralFillStealthHover};
+            color: ${neutralForegroundHover};
+        }
 
-    :host(:active) {
-        color: ${neutralForegroundRest};
-        fill: currentcolor;
-    }
+        :host(:not(.disabled):active),
+        :host(.vertical:not(.disabled):active) {
+            background: ${neutralFillStealthActive};
+            color: ${neutralForegroundActive};
+        }
 
-    :host([disabled]) {
-        cursor: ${disabledCursor};
-        opacity: ${disabledOpacity};
-    }
+        :host([aria-selected="true"]),
+        :host(.vertical[aria-selected="true"]) {
+            z-index: 2;
+            background: ${accentFillRest};
```

Comment:
Let's make this like listbox option or tree view item. Selected state should be the accent border with neutral fill.

Reply:
updated